src/shared/common/components/side-bar.jsx

```diff
@@ -59,19 +61,25 @@ class SideBar extends Component {
         );
     }
 
+    updateTabs() {
+        const { page, dispatch } = this.props;
+        if (page && page.params && page.params.caseId) {
+            updateCaseConfig(page.params.caseId, dispatch);
+        }
+    }
+
     render() {
-        const { type } = this.props.summary;
-        const caseTabs = this.resolveTabsForCaseTypeByShortCode(type);
+        const caseTabs = this.props.caseConfig ? this.props.caseConfig.tabs : ['documents'];
         return (
             <Fragment>
                 <div className='tabs'>
                     <ul>
-                        {this.renderTabButton('Documents', 'DOCUMENTS')}
-                        {this.renderTabButton('Summary', 'SUMMARY')}
-                        {this.renderTabButton('Timeline', 'TIMELINE')}
-                        {caseTabs.people && this.renderTabButton('People', 'PEOPLE')}
-                        {caseTabs.case_actions && this.renderTabButton('Actions', 'CASE_ACTIONS')}
-                        {caseTabs.ex_gratia && this.renderTabButton('Ex-Gratia', 'EX_GRATIA')}
+                        {caseTabs.includes('documents') && this.renderTabButton('Documents', 'DOCUMENTS')}
+                        {caseTabs.includes('summary') && this.renderTabButton('Summary', 'SUMMARY')}
+                        {caseTabs.includes('timeline') && this.renderTabButton('Timeline', 'TIMELINE')}
+                        {caseTabs.includes('people') && this.renderTabButton('People', 'PEOPLE')}
+                        {caseTabs.includes('actions') && this.renderTabButton('Actions', 'FOI_ACTIONS')}
+                        {caseTabs.includes('ex_gratia') && this.renderTabButton('Ex-Gratia', 'EX_GRATIA')}
```

Comment:
By default on line 19 the active tab is set to documents if there is no active tab. Would be worth checking what happens if documents is not a tab allowed on the case type being viewed

Reply:
Good spot, it renders the document tab regardless. I think it should probably default to the left-most tab instead

Reply:
I've changed it so that it will pick the leftmost tab as default rather than just documents